A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -32,6 +32,76 @@
             </UseTemplate>
         </Template>
 
+        <Template Name="A32NX_HANDLING_Push_Autobrakes_Template">
```

Comment:
Please put this template in `A32NX_interior.xml` and prefix it with `FBW_Airbus` instead of `A32NX`. Thank you !

Reply:
Also, if you're at it, you could also do the same with the A32NX_LANDING_GEAR_Light_Template.

Reply:
Will do, so it can be merged in one go